Added python-dev as a pre-req for Linux-quickstart

saki-osive opened the pull request:
While setting up a local instance using the Linux quickstart page, the ```python-dev``` dependency is missing from the list of pre-requisites which is needed. This PR adds the ```python-dev``` dependency to the documentation.

sharifsalah commented:
Thanks @saki-osive. I don't recall needing this package the last time I ran through the quick start, but it might be a new dependency. Hopefully @bcipriano can confirm.

saki-osive commented:
Hi, thanks for your reply, can you recall for which version of Python was it for, as for many version I have tried it on, required this dependency?

bcipriano commented:
I'm a bit confused here -- the link given for `python-dev` seems to point to the PyPi page for the Flask library? I can't really think of why we would need Flask for this process. I could be wrong.

Are you sure you don't mean we need the `python-dev` *package*? I.e. `apt get install python-dev` (on Ubuntu/Debian) or `yum install python-devel` (on e.g. CentOS).

saki-osive commented:
Hi, I exactly mean the ```python-dev``` dependency and not the ```Flask``` library. As you have correctly pointed out, the link I added was misleading, therefore I have removed it. Please verify this now.

bcipriano commented on content/docs/Quick starts/quick-start-linux.md:
````diff
@@ -27,6 +27,7 @@ You must have the following software installed on your machine:
 *   Python version 2.7 or greater
 *   The Python [`pip` command](https://pypi.org/project/pip/)
 *   The Python [virtualenv tool](https://pypi.org/project/virtualenv/)
+*   The ```python-dev``` command 
````
I think we should refer to this as a "package". Calling it a command suggests that you can run `python-dev` in a shell to check if it's installed, which is not the case if I understand correctly.

I would also mention that this is named `python-devel` on some Linux distributions.

saki-osive replied:
Hi, thanks, I have updated, please verify.

bcipriano commented on content/docs/Quick starts/quick-start-linux.md:
````diff
@@ -40,6 +41,9 @@ must do one of the following:
 
         git clone https://github.com/AcademySoftwareFoundation/OpenCue.git
+    
+3. Make sure you have the ```python-dev package``` installed on your system corresponding to the Python version you
````
I don't think we need this added step here -- if we've mentioned it in the "Before you begin" section, that's good enough, it's just redundant here.

saki-osive replied:
I have removed the redundant step in the new commit, please verify, thanks.

bcipriano reviewed (changes requested):
Thanks!

saki-osive commented:
@bcipriano @sharifsalah Please verify the latest commit. Thanks.

sharifsalah commented on content/docs/Quick starts/quick-start-linux.md:
````diff
@@ -27,6 +27,7 @@ You must have the following software installed on your machine:
 *   Python version 2.7 or greater
 *   The Python [`pip` command](https://pypi.org/project/pip/)
 *   The Python [virtualenv tool](https://pypi.org/project/virtualenv/)
+*   The ```python-dev``` (Ubuntu/Debian) or ```python-devel```(Fedora/Cent-OS) package 
````
Generally it's best to avoid parentheses as readers tend to ignore information in them. You could rewrite this bullet point something like:

*   If you're using Ubuntu or Debian, the `python-dev` package
*   If you're using Fedora or Cent-OS, the `python-devel` package

saki-osive replied:
Thanks, I have reformatted it conforming to the said format, please verify it.

saki-osive commented:
Thanks, I have reformatted the bullet points, please verify.

sharifsalah reviewed (approved):
Looks good @saki-osive, thanks.
